lua/astrocommunity/motion/mini-surround/init.lua

```diff
@@ -7,6 +7,7 @@ return {
   { "machakann/vim-sandwich", enabled = false },
   {
     "echasnovski/mini.surround",
+    event = "User AstroFile",
```

Comment:
```suggestion
    event = { "User AstroFile", "InsertEnter" }
```

@Uzaaft this would be a good change